packages/story-editor/src/components/canvas/shoppingPageAttachment/index.js

```diff
@@ -100,17 +101,18 @@ function PageAttachment({ theme }) {
           <ArrowBar fill={arrowColor} />
         </ArrowWrap>
         <OutlinkChip>
-          <TextWrapper fgColor={fgColor}>
-            {__('Shop Now', 'web-stories')}
-          </TextWrapper>
+          <TextWrapper fgColor={fgColor}>{ctaText}</TextWrapper>
         </OutlinkChip>
       </>
     </Wrapper>
   );
 }
 
 PageAttachment.propTypes = {
-  theme: PropTypes.string,
+  pageAttachment: PropTypes.shape({
+    ctaText: PropTypes.string,
+    theme: PropTypes.string,
+  }),
```

Comment:
As per my other two suggestions for the props:

```suggestion
  ctaText: PropTypes.string,
  theme: PropTypes.string,
```